pkg/app/utils/utils.go

```diff
@@ -33,7 +33,8 @@ func EnsureAppProjectName(userNSClient v1.NamespaceInterface, ownedProjectID, cl
 	} else {
 		deployNamespace = &corev1.Namespace{
 			ObjectMeta: metav1.ObjectMeta{
-				Name: appTargetNamespace,
+				Name:        appTargetNamespace,
+				Annotations: map[string]string{"field.cattle.io/creatorId": creatorID},
```

Comment:
Use constant for the annotation key. Check if an existing one is appliable

Reply:
It works fine